daemon/list.go

```diff
@@ -28,13 +30,15 @@ type ContainersConfig struct {
 
 func (daemon *Daemon) Containers(config *ContainersConfig) ([]*types.Container, error) {
 	var (
-		foundBefore bool
-		displayed   int
-		all         = config.All
-		n           = config.Limit
-		psFilters   filters.Args
-		filtExited  []int
+		foundBefore    bool
+		displayed      int
+		ancestorFilter bool
+		all            = config.All
+		n              = config.Limit
+		psFilters      filters.Args
+		filtExited     []int
 	)
+	imagesFilter := map[string]struct{}{}
```

Comment:
@calavera Yeah I was wondering about that : "empty struct + extra `ok`" or a "`bool` and nothing else". The later takes a little bit more memory but probably not enough to not do it ; I think.